### PR TITLE
Remove custom criteria naming code and remove remaining references

### PR DESCRIPTION
**Pull request by tjennison-google**
This also improves concept set naming.

### CODE REVIEW

File: ui/src/cohort.ts [hunk 1]
```diff
@@ -61,6 +61,14 @@ export type DisplayDetails = {
   additionalText?: string[];
 };
 
+export function criteriaTitle<DataType>(
```
Comment (kemp-verily) [inline ui/src/cohort.ts:64]: optional, but i would use a verb prefix here (eg `get`)

**Review by kemp-verily — COMMENTED** [review]
lg for react/typescript with a nit

File: ui/src/cohort.ts (hunk 1 above)
Comment (tjennison-google) [reply ui/src/cohort.ts]: Done.

**Review by marikomedlock — COMMENTED** [review]
What's the reason behind removing the ability to name criteria groups and concept sets? Code changes look fine.

**Comment by tjennison-google** [comment]
> What's the reason behind removing the ability to name criteria groups and concept sets? Code changes look fine.

I'm moving the UI to single select instances within a criteria. Each criteria can then be automatically named based on the actual concept that is selected so there isn't a need to override it like with "Contains Condition Codes".

**Comment by marikomedlock** [comment]
> I'm moving the UI to single select instances within a criteria. Each criteria can then be automatically named based on the actual concept that is selected so there isn't a need to override it like with "Contains Condition Codes".

That makes sense for single/binary filters (condition=diabetes). Could the generated names get unwieldy for array filters (condition=diabetes and ethnicity=japanese and ...)?

**Comment by tjennison-google** [comment]
> > I'm moving the UI to single select instances within a criteria. Each criteria can then be automatically named based on the actual concept that is selected so there isn't a need to override it like with "Contains Condition Codes".
> 
> That makes sense for single/binary filters (condition=diabetes). Could the generated names get unwieldy for array filters (condition=diabetes and ethnicity=japanese and ...)?

That would be groups, which can still be renamed. This is just the case for `condition=(diabetes or cancer)` which would instead be `condition=diabetes or condition=cancer`. I need to make a better habit of including screenshots in these PRs.

image

**Review by marikomedlock — APPROVED** [review]
nit: In the PR description, do you mean "removes" instead of "improves"?

**Comment by tjennison-google** [comment]
> nit: In the PR description, do you mean "removes" instead of "improves"?

I did actually mean "improves". They had bad default names and no ability to rename them. I've edited the description to clarify.

**Pull request merged**